### PR TITLE
Typo correctness for some UI files

### PR DESCRIPTION
**Pull request by shrinktofit**

### CODE REVIEW

**Comment by shrinktofit** [comment]
Request review from @JoneLau

File: cocos/ui/assembler/label/ttf.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const ttf: IAssembler = {
 
         vbuf.set(vData, vertexOffset);
 
-        // fill indice data
+        // fill index data
```
Comment (JoneLau) [inline cocos/ui/assembler/label/ttf.ts]: indices

File: cocos/ui/assembler/sprite/simple.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ export const simple: IAssembler = {
 
         vbuf.set(vData, vertexOffset);
 
-        // fill indice data
+        // fill index data
```
Comment (JoneLau) [inline cocos/ui/assembler/sprite/simple.ts]: indices

File: cocos/ui/assembler/utils.ts [hunk 3]
```diff
@@ -87,7 +87,7 @@ export function fillMeshVertices3D (node: INode, renderer: UI, renderData: Rende
         vertexOffset += 4;
     }
 
-    // fill indice data
+    // fill index data
```
Comment (JoneLau) [inline cocos/ui/assembler/utils.ts]: indices

File: cocos/ui/assembler/graphics/webgl/earcut.ts [hunk 4]
```diff
@@ -26,14 +26,14 @@
...
-    // vertice index in coordinates array
+    // vertex index in coordinates array
     public i: number;
 
     // vertex coordinates
     public x: number;
     public y: number;
 
-    // previous and next vertice nodes in a polygon ring
+    // previous and next vertex nodes in a polygon ring
```
Comment (JoneLau) [inline cocos/ui/assembler/graphics/webgl/earcut.ts]: vertices?

**Review by JoneLau — APPROVED** [review]

**Comment by pandamicro** [comment]
尽快修复一下冲突 @shrinktofit

**Pull request merged**